app/components/ChartSettings/modules/Metadata.js

```diff
@@ -39,10 +51,13 @@ export default class Metadata extends Component {
   };
 
   render() {
+    if (!this.shouldShowMetadata.subtitle) {
+      delete this.state.subtitle;
```

Comment:
Yeah it's weird about ternary statements as the implicit return of an arrow function. Implicit return is just a parlor trick anyway, this is fine the way you have it. 😄